v20.1/admin-ui-cluster-overview-page.md

```diff
@@ -3,88 +3,131 @@ title: Cluster Overview Page
 toc: true
 ---
 
-The **Cluster Overview** page of the Admin UI provides details of the cluster nodes and their liveness status, replication status, uptime, and key hardware metrics. [Enterprise users](enterprise-licensing.html) can enable and switch to the [Node Map](admin-ui-cluster-overview-page.html#node-map-enterprise) view.
+The **Cluster Overview** page of the Admin UI displays key metrics about your cluster and individual nodes. These include liveness status, replication status, uptime, and hardware usage.
 
-## Cluster Overview Panel
+If you have an [enterprise license](enterprise-licensing.html), you can enable the [Node Map](#node-map-enterprise) view for a visual representation of your cluster.
```

Comment:
I might add "for a visual representation of your cluster's geographic layout" or something similar.

This could be a green "tip" callout, but that's more of a subjective call.